about.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Plug about</title>
+    <link rel="stylesheet" href="about.css">
+</head>
+<body>
+    <section class="start">
+        <header class="realHead">
+            <div class="navLine">
+                <nav>
+                    <div class="sideMback">
+                        <img class="sideMenu" onclick="openMobileMenu()" src="img/sideMenu.png" alt="">
+                    </div>
+                    <div class="navSlide">
+                        <div class="mediNbac">
+                            <ul class="mediaNav">
+                                <li><img src="img/facebook.png" alt=""></li>
+                                <li><img src="img/twitter.png" alt=""></li>
+                                <li>English</li>
+                                <li>My page</li>
+                                <li>Login</li>
+                            </ul>
```

Comment:
**Changes Suggested**
- [ ] The icons for facebook and twitter are not properly aligned on top. Please let us add some padding and vertically center them.

Reply:
fixed